### PR TITLE
Helm Auth Config with updated console

### PR DESCRIPTION
**Pull request by acohen4**

### CODE REVIEW

File: etc/helm/examples/local-dev-values.yaml [hunk 1]
```diff
@@ -14,19 +14,11 @@ pachd:
   metrics:
     enabled: false
   clusterDeploymentID: dev
-  ## if pachd.enterpriseLicenseKey is set, the following values will be interpreted to setup Auth
-  # enterpriseLicenseKey: ""
-  oauthIssuer: "http://pachd:1658"
-  oauthRedirectURI: "http://localhost:30657/authorization-code/callback"
-  userAccessibleOauthIssuerHost: "localhost:30658"
-  mockIDP: true
+  activateEnterprise: false
```
Comment (acohen4) [inline etc/helm/examples/local-dev-values.yaml:17]: add this to doc....

File: etc/helm/pachyderm/templates/pachd/config-secret.yaml [hunk 2]
```diff
@@ -24,26 +24,28 @@ stringData:
...
+    {{- end }}
+    redirect_uri: {{ include "pachyderm.pachdRedirectURI" . }}
     scopes:
     - email
     - profile
     - groups
     - openid 
 
-{{ if eq .Values.deployTarget "LOCAL" }}
+  # TODO: When should this _not_ be specified?
```
Comment (acohen4) [inline etc/helm/pachyderm/templates/pachd/config-secret.yaml:42]: I think I can remove this entirely. It's supposed to serve as an example for pushing pachyderm Cluster Role bindings to pachd

**Review by chainlink — CHANGES_REQUESTED** [review]
First pass

**Comment by chainlink** [comment]
Looks like we can totally eliminate the local-values.yaml file with this PR 🎉

**Comment by chainlink** [comment]
Need to change the check in etc/helm/pachyderm/templates/pachd/config-job.yaml to `activateEnterprise` instead of `enterpriseLicenseKey`

File: etc/helm/pachyderm/templates/pachd/config-secret.yaml (hunk 2 above)
Comment (acohen4) [reply etc/helm/pachyderm/templates/pachd/config-secret.yaml:42]: yeah sure. How does having this piece in a comment block sound?

File: etc/helm/pachyderm/values.yaml [hunk 3]
```diff
@@ -53,15 +53,15 @@ console:
...
     #  memory: "2G"
   config:
-    issuerURI: "" # Cannot have trailing slash
     reactAppRuntimeIssuerURI: "" # Inferred if running locally or using ingress
     oauthRedirectURI: "" # Infered if running locally or using ingress
-    oauthClientID: "dash"
+    oauthClientID: "console"
     oauthClientSecret: "" # Autogenerated on install if blank
     graphqlPort: 4000
     oauthPachdClientID: pachd
```
Comment (chainlink) [inline etc/helm/pachyderm/values.yaml:61]: `oauthPachdClientID` should be removed

**Review by chainlink — CHANGES_REQUESTED** [review]
One small change, but otherwise looks good

**Comment by chainlink** [comment]
Can you add docs to both the helm docs and to the values.yaml file for the new fields?

**Pull request merged**